snapshot/cli/cli.go

```diff
@@ -0,0 +1,84 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/scootdev/scoot/snapshot"
+	"github.com/scootdev/scoot/snapshot/git/repo"
+)
+
+type DBInjector interface {
+	RegisterFlags(rootCmd *cobra.Command)
+	Inject() (snapshot.DB, error)
+}
+
+func MakeDBCLI(injector DBInjector) *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "scoot-snapshot-db",
+		Short: "scoot snapshot db CLI",
+	}
+	parentCmd := rootCmd
+	add := func(subCmd dbCommand) {
+		cmd := subCmd.register()
```

Comment:
Kind of confusing to switch between dbCommand and cobra.Command without tagging the vars as such.